pretty/src/main/java/org/aya/pretty/doc/Doc.java

```diff
@@ -227,6 +227,11 @@ record PageWidth(@NotNull IntFunction<Doc> docBuilder) implements Doc {
     return cat(symbol(leftSymbol), doc, symbol(rightSymbol));
   }
 
+  /** @param explicit when false, add braces */
+  static @NotNull Doc braced(Doc doc, boolean explicit) {
```

Comment:
Since it's in the pretty module. I suggest rename to `bracedIfNot`

Reply:
`static @NotNull Doc bracedUnless(Doc doc, boolean falsification)`